tsconfig.json

```diff
@@ -1,26 +1,20 @@
 {
-  "compilerOptions": {
-    "target": "es5",
-    "lib": [
-      "dom",
-      "dom.iterable",
-      "esnext"
-    ],
-    "allowJs": true,
-    "skipLibCheck": true,
-    "esModuleInterop": true,
-    "allowSyntheticDefaultImports": true,
-    "strict": true,
-    "forceConsistentCasingInFileNames": true,
-    "noFallthroughCasesInSwitch": true,
-    "module": "esnext",
-    "moduleResolution": "node",
-    "resolveJsonModule": true,
-    "isolatedModules": true,
-    "noEmit": true,
-    "jsx": "react-jsx"
-  },
-  "include": [
-    "src"
-  ]
+	"compilerOptions": {
+		"target": "es5",
+		"lib": ["dom", "dom.iterable", "esnext"],
+		"allowJs": true,
+		"skipLibCheck": true,
+		"esModuleInterop": true,
+		"allowSyntheticDefaultImports": true,
+		"strict": true,
+		"forceConsistentCasingInFileNames": true,
+		"noFallthroughCasesInSwitch": true,
+		"module": "esnext",
+		"moduleResolution": "node",
+		"resolveJsonModule": true,
+		"isolatedModules": true,
+		"noEmit": true,
+		"jsx": "react-jsx"
+	},
+	"include": ["src", "test"]
```

Comment:
I think this is fine but we need to confirm that updating this does not cause the `test` to be included in the final artifact (produced by `npm run build`)

Reply:
Who do I do that?

Reply:
1. With `test` in the `include`, run `npm run build`
2. Navigate to the produced `dist` directory
3. Verify no `test` dir is in the `dist` directory - when app is deployed, `dist` is the directory (and its contents) that will be hosted online. If `test` code is up there, we are inflating our app size and shipping test code (both of which are not great, xD)

Reply:
I used the following command in PowerShell - 

`cls; npm i; npm run build`

`cls` - clear screen
`npm i` - install NodeJS deps
`npm run build` - produces directory ready for deployment

Looks like by default, our output dir is named `build` (not `dist`), but the rest of the checks still follow. That being said, it can be a bit mundane to look through the generated files, so I just searched for docs on the TSConfig file and found some info on it -

https://www.typescriptlang.org/tsconfig#include

Looks like that property simply configures how the compiler will look for files/sources and does not affect build output (in our case, create-react-app is what configures the build output).

TLDR; - this change is fine